README.md

```diff
@@ -22,23 +22,40 @@ Or install it yourself as:
 
 # Usage
 
-The easiest entry point into the library is the `Sync` class, which will handle building, validating, and performing all the necessary operations to sync an object.  You can of course build and manage these primitives yourself if you require more control.  Here is an example of a sync:
+To use LedgerSync, you must create an `Operation`.  The operation will be ledger-specific, so it will require the following:
+
+1. Adaptor
+2. Resource(s)
+
+The code may look like the following:
 
 ```ruby
-sync = LedgerSync::Sync.new(
-  adaptor: adaptor, # The adaptor for the ledger you want to use (see below)
-  method: :upsert,
-  resources_data: resources_data, # The structured hash of resource data (see below)
-  resource_external_id: resource_external_id, # The root object ID on which the operation is to be performed
-  resource_type: resource_type # The root object type on which the operation is to be performed
+adaptor = LedgerSync::Adaptors::QuickBooksOnline::Adaptor.new(
+  access_token: ENV['QUICKBOOKS_ONLINE_ACCESS_TOKEN'],
```

Comment:
This probably wouldn't be an ENV variable, since it can change right? Just thinking in terms of the example. You could do a line like
```
access_token: access_token # assuming it's defined somewhere above this snippet
```

Reply:
Same for refresh token below